themes/delphi/layouts/partials/nav.html

```diff
@@ -1,9 +1,58 @@
-<nav class="uk-navbar-container" data-uk-navbar>
+<nav class="uk-visible@s uk-navbar-container" data-uk-navbar>
   <div class="uk-navbar-left">
       <ul class="uk-navbar-nav">
-          {{ range .Site.Menus.main.ByWeight }}
-          <li class="{{ if eq .Identifier "main" }}uk-active{{ end }}"><a href="{{ .URL | relLangURL }}">{{ .Name }}</a></li>
-          {{ end }}
+        <li>
+          {{partial "delphi-text-logo.html"}}
+        </li>
+        {{ $currentPage := . }}
+        {{ range .Site.Menus.main.ByWeight }}
+        <li class="{{ if (eq $currentPage.URL .URL) }}active{{ end }}">
```

Comment:
have you tried using https://gohugo.io/functions/hasmenucurrent/? instead

Reply:
Yes, that wasn't working for me, and I looked at some other posts and saw that it seems not to work without putting the menu items in the "Front Matter" for the pages, and I saw this used as a workaround